### ISSUE TITLE
Focus goes to top of the page after going through "Learn more about how it works" screens AND "The app will now ask your permission" screen

### ISSUE THREAD

**Issue by qanitah** [cds-snc/covid-alert-app#689]
With TalkBack on, when the user goes from onboarding screen step 4  "How it works" to "learn more about how it works" and then clicks on Done or Close, the focus goes back to top of the visual screen when you right swipe for next element. On portrait, it goes to image on top while in landscape, it goes to the text after the heading. The focus should be at the "Learn more about how it works" button so the user doesn't have to listen to the whole page again.

Please see screen shots to see where the focus goes.

With iOS, it goes back to top of the screen at "Step 4 of 6" (in both portrait and landscape). 

It's the same for "Step 5 of 6" when user returns from "Learn more about privacy" screen.

Android 8.1.0 on Samsung Galaxy
iOS 13.5.1 on iPhone 7

![image](https://user-images.githubusercontent.com/14280062/87469749-c0bec100-c5e9-11ea-8769-a9e242b4d0a8.png)

![image](https://user-images.githubusercontent.com/14280062/87469204-e8f9f000-c5e8-11ea-8c4d-f1380f5e0289.png)

![image](https://user-images.githubusercontent.com/14280062/87469726-b7355900-c5e9-11ea-8d7b-112e4575e976.png)

**Comment by timarney** [comment]
Closing this - needs to be re-tested on latest version + latest OS version.  Can re-open as needed.
